command/agent/job_endpoint.go

```diff
@@ -270,8 +270,10 @@ func (s *HTTPServer) jobDeployments(resp http.ResponseWriter, req *http.Request,
 	if req.Method != "GET" {
 		return nil, CodedError(405, ErrInvalidMethod)
 	}
+	all, _ := strconv.ParseBool(req.URL.Query().Get("all"))
```

Comment:
is the behavior of ParseBool for an empty string the only thing that makes all default to false? If so, maybe that needs a comment? It's builtin to go, maybe it doesn't